packages/venia-ui/lib/targets/venia-ui-declare.js

```diff
@@ -3,93 +3,159 @@
  *
  * Their implementations are found in `./venia-ui-intercept.js`.
  *
- * @module VeniaUI/Targets
  */
 module.exports = targets => {
     targets.declare({
         /**
-         * A file that implements the RichContentRenderer interface.
+         * Provides access to the list of rendering strategies used by the
+         * RichContent component.
          *
-         * @typedef {Object} RichContentRenderer
-         * @property {string} componentName - Friendly name for the React
-         *   component, for debugging purposes.
-         * @property {string} importPath - Path to the implementation file.
-         *   Can be anything the resolves with an `import` statement.
-         */
-
-        /**
-         * Registers RichContentRenderers by putting them in an array.
+         * This target collects a list of RichContentRenderer modules.
+         * It builds an array of these renderers, which Venia's RichContent
+         * component uses to try and render a block of "rich" content, such
+         * as HTML.
          *
-         * @typedef {Object} RichContentRendererCollection
-         * @method add - Register a RichContentRenderer.
-         * @param {RichContentRenderer} renderer
-         */
-
-        /**
-         * @callback rendererIntercept
-         * @param {RichContentRendererCollection} renderers
-         * @returns {undefined} - Interceptors of `richContentRenderers` can
-         *   call `renderers.add()` and not return anything.
-         */
-
-        /**
-         * Collects RichContentRenderers contributed by third party extensions.
-         * Builds an array of these renderers which Venia's RichContent
-         * component uses to try and render a block of "rich" content, most
-         * likely HTML.
+         * Use this target if your backend system uses a customized content
+         * storage format instead of plain HTML in "rich content" fields such
+         * as product descriptions and CMS blocks.
```

Comment:
This is a slight inaccuracy as currently product descriptions use the RichText component for displaying descriptions
https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.js#L167

Reply:
This is describing the fields in our API responses that may contain "rich content", not the frontend's ability to display them. It's true that our ProductFullDetail component doesn't use `<RichContent>` and we should fix that, but I think here we are talking about backend capabilities and not frontend.